.eslintrc.json

```diff
@@ -11,6 +11,7 @@
   },
   "plugins": ["@typescript-eslint"],
   "rules": {
+    "@typescript-eslint/ban-ts-comment": "off", // Should be enabled after TODOs are resolved
```

Comment:
Instead of having it off, you can just temporarily suppress TODO errors in that file

Reply:
Actually, that comment is somehow misleading. The line is to turn off the rule for `@ts-ignore` which if we don't turn off gives us an error from the linter. The part for the TODO errors is the rule `no-warning-comments` which is also commented on line 50

Reply:
Then suppress ts-ignore in that file

Reply:
https://github.com/Shard-Labs/starknet-hardhat-example/pull/74/commits/9113384642c706ac1504067dac078d89c7a0857c